readthedocs/oauth/migrations/0013_data_migration_for_remote_relation_model.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:15
+
+import json
+from itertools import islice
+
+from django.db import migrations
+
+
+def move_data_to_remote_relations(apps, schema_editor):
+    RemoteRelation = apps.get_model('oauth', 'RemoteRelation')
+
+    def remote_relations_generator(relations, batch_size):
+        for relation in relations.iterator(chunk_size=batch_size):
+            relation.account_id = relation.remoterepository.account_id
```

Comment:
Shouldn't be `relation.account` here?

Reply:
We only need to make the ``ForeinKey`` relation here, so we can just use the ``_id`` to directly reference the ID and store that in the ``RemoteRelation`` model. It will make it so that no extra query is made to retrieve the ``account`` object and we don't need to add it to ``select related()`` also.

https://docs.djangoproject.com/en/3.1/ref/models/fields/#database-representation